Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Configurations/TouchActionsSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Aquality.Selenium.Core.Configurations;
+
+namespace Aquality.Appium.Mobile.Configurations
+{
+    public class TouchActionsSettings : ITouchActionsSettings
+    {
+        private readonly ISettingsFile settingsFile;
+        private readonly string swipeSettingsPath = ".touchActions.swipe";
+
+        public TouchActionsSettings(ISettingsFile settingsFile)
+        {
+            this.settingsFile = settingsFile;
+        }
+
+        public int SwipeRetries => settingsFile.GetValue<int>($"{swipeSettingsPath}.retries");
+
+        public long SwipeDuration => settingsFile.GetValue<long>($"{swipeSettingsPath}.duration");
```

Comment:
please use TimeSpan for durations. TimeSpan.FromSeconds(settingsFile.GetValue<long>($"{swipeSettingsPath}.duration"))